### PR TITLE
Move change log to CHANGELOG.md

### PR DESCRIPTION
**Pull request by Flanker32**
Move change log to CHANGELOG.md, for issue #2909

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -33,333 +33,11 @@
...
         <li><a href="https://github.com/Microsoft/azure-tools-for-java/issues/2714" rel="nofollow">#2714</a>,<a href="https://github.com/Microsoft/azure-tools-for-java/issues/2688" rel="nofollow">#2688</a>,<a href="https://github.com/Microsoft/azure-tools-for-java/issues/2669" rel="nofollow">#2669</a>,<a href="https://github.com/Microsoft/azure-tools-for-java/issues/2728" rel="nofollow">#2728</a>,<a href="https://github.com/Microsoft/azure-tools-for-java/issues/2807" rel="nofollow">#2807</a>,<a href="https://github.com/Microsoft/azure-tools-for-java/issues/2808" rel="nofollow">#2808</a>,<a href="https://github.com/Microsoft/azure-tools-for-java/issues/2811" rel="nofollow">#2811</a>,<a href="https://github.com/Microsoft/azure-tools-for-java/issues/2831" rel="nofollow">#2831</a>Spark Run Configuration validation issues.</li>
         <li><a href="https://github.com/Microsoft/azure-tools-for-java/issues/2810" rel="nofollow">#2810</a>,<a href="https://github.com/Microsoft/azure-tools-for-java/issues/2760" rel="nofollow">#2760</a> Spark Run Configuration issues when created from context menu.</li>
```
Comment (wezhang) [inline PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml]: Can we have a line at bottom to link to the `CHANGELOG.md`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,317 @@
+# Change Log
+
+All notable changes to "Azure Toolkit for IntelliJ IDEA" will be documented in this file.
+
+- [Change Log](#change-log)
+
+  - [[3.19.0]](#3190)
```
Comment (wezhang) [inline CHANGELOG.md]: How about to remove `[ ]` around the version? it looks weird.

File: PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml (hunk 1 above)
Comment (Flanker32) [reply PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml]: yes, added a link at the bottom of this page

File: CHANGELOG.md (hunk 2 above)
Comment (Flanker32) [reply CHANGELOG.md]: Done~

**Pull request merged**